### ISSUE TITLE
Feature Request: Adding a new webapp war using a maven dependency

### ISSUE THREAD

**Issue by toonconvents** [Alfresco/alfresco-sdk#412]
Add the possibility to inject another webapp war into the tomcat maven plugin.
Ideally with the freedom to configure both contextPath and contextFile properties for this webapp.

We have an application that has Alfresco at it's core but comprises another essential webapp too. This webapp works in concert with Alfresco and so is essential when developing for Alfresco. Our current config (pre-v3) allows us to add this directly in the maven tomcat plugin like so:

```
<webapp>
    <groupId>com.exari</groupId>
    <artifactId>exari-docgen-cmwar</artifactId>
    <version>${project.version}</version>
    <type>war</type>
    <asWebapp>true</asWebapp>
    <contextPath>/exari</contextPath>
    <contextFile>${project.build.directory}/contexts/context-docgen.xml</contextFile>
</webapp>
```

This would facilitate our development process a lot from v3 onwards.
Thanks

**Comment by gravitonian** [comment]
This is now available if building latest SNAPSHOT:

```
<plugin>
                <groupId>org.alfresco.maven.plugin</groupId>
                <artifactId>alfresco-maven-plugin</artifactId>
                <version>${alfresco.sdk.version}</version>
                <configuration>
....
                    <tomcatCustomWebapps>
                        <tomcatCustomWebapp>
                            <groupId>org.alfresco</groupId>
                            <artifactId>api-explorer</artifactId>
                            <version>1.2</version>
                            <contextPath>/api-explorer</contextPath>
                            <contextFile></contextFile>
                        </tomcatCustomWebapp>
                    </tomcatCustomWebapps>
```

**Comment by ohej** [comment]
Fixed in a07219e6e6a5a9ccfa03d06b081da21434430aef
